doc/changelog.rst

```diff
@@ -6,6 +6,22 @@ Changes in Version 4.8.0
 
 .. warning:: PyMongo 4.8 drops support for Python 3.7 and PyPy 3.8: Python 3.8+ or PyPy 3.9+ is now required.
 
+Changes in Version 4.7.2
+-------------------------
+
+Version 4.7.2 fixes a bug introduced in 4.7.0:
```

Comment:
This is ambiguous. Is there just the Nuitka compiler bug?

Reply:
Yes

Reply:
How about just one line:

Fixes a bug where PyMongo could not be used with the Nuitka compiler.

Reply:
I want to convey that the bug was introduced in 4.7.0.  It worked in prior versions.